tests/phpunit/integration/tests/Admin/Dashboard.php

```diff
@@ -158,6 +158,27 @@ public function test_enqueue_assets_wrong_page() {
 		$this->assertFalse( wp_style_is( $dashboard::SCRIPT_HANDLE ) );
 	}
 
+	/**
+	 * @covers ::get_post_type_archive_link
+	 * @covers \Google\Web_Stories\Traits\Post_Type::get_post_type_has_archive
+	 */
+	public function test_get_post_type_archive_link() {
+		$dashboard = new \Google\Web_Stories\Admin\Dashboard(
+			$this->createMock( \Google\Web_Stories\Experiments::class ),
+			$this->createMock( \Google\Web_Stories\Integrations\Site_Kit::class ),
+			$this->createMock( \Google\Web_Stories\Decoder::class ),
+			$this->createMock( \Google\Web_Stories\Locale::class ),
+			( new \Google\Web_Stories\Admin\Google_Fonts() ),
+			( new \Google\Web_Stories\Assets() )
+		);
+
+		$result = $this->call_private_method( $dashboard, 'get_post_type_archive_link', [ \Google\Web_Stories\Story_Post_Type::POST_TYPE_SLUG ] );
+		$this->assertSame( get_post_type_archive_link( \Google\Web_Stories\Story_Post_Type::POST_TYPE_SLUG ), $result );
+
+		$result = $this->call_private_method( $dashboard, 'get_post_type_archive_link', [ \Google\Web_Stories\Template_Post_Type::POST_TYPE_SLUG ] );
+		$this->assertNotSame( get_post_type_archive_link( \Google\Web_Stories\Template_Post_Type::POST_TYPE_SLUG ), $result );
```

Comment:
A bit confusing to test this function with `Template_Post_Type`. Just because it doesn't have an archive by default?

Reply:
> Just because it doesn't have an archive by default?

Yeah it is. Should I just create two new post types for this test?

Reply:
That might work, yeah 👍